src/Morphir/Elm/IncrementalFrontend.elm

```diff
@@ -1130,3 +1144,127 @@ extractValues resolveValueName parsedModule =
                     )
     in
     orderedDeclarationAsDefinitions
+
+
+{-| Returns a Set of implicitly exposed modules.
+A module `ModA` is implicitly exposed if an explicitly exposed module `ModB` uses one of it's type in a public interface.
+This could be as a result of a public type in `ModB` dependending on a type in `ModA` or a public function defined in
+`ModB` that takes, as an input, a type defined in `ModA`.
+
+The parameters are
+
+  - moduleDef - All the modules
+  - exposedModules - A set of exposed module-names
+
+-}
+collectImplicitlyExposedModules : PackageName -> Dict Path (Module.Definition ta va) -> Set Path -> Set Path
```

Comment:
Did you copy-paste this from the `Frontend`? If so, I would reference this one from the `Frontend` instead since we are going to remove that one at some point and only this one will be kept.